app/src/main/java/com/keylesspalace/tusky/service/SendStatusService.kt

```diff
@@ -181,6 +182,25 @@ class SendStatusService : Service(), Injectable {
                 return@launch
             }
 
+            val isEditing = statusToSend.statusId != null
```

Comment:
You could refactor the `createStatus`/`editStatus` block below to reuse this variable 🙂